### DISCUSSION TITLE
image limitedRemainingText not working

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#8660] [Help]
in the table, the image column is not showing the limitedRemainingText

this is my code:

ImageColumn::make('images')
->circular()
->stacked()
->limit(1)
->limitedRemainingText(isSeparate: true)

thx in advanced

**Comment by workflou** [comment]
same here

**Reply by leandrocfe** [reply]
What isn't working? This is working for me.
Could you try this code please?

```php
ImageColumn::make('images')
    ->getStateUsing(fn (): array => [
        'https://i.pravatar.cc/50',
        'https://i.pravatar.cc/50',
        'https://i.pravatar.cc/50',
        'https://i.pravatar.cc/50',
        'https://i.pravatar.cc/50',
    ])
    ->circular()
    ->stacked()
    ->limit(3)
    ->limitedRemainingText(isSeparate: true),
```

**Reply by jcgmalave** [reply]
it work if the ->limit(3) parameter is equal or greater than 3, i try with 1 and 2 and it doesnt work

**Reply by Ettemlevest** [reply]
above example code does not display the remaining text count for me.
<img width="120" alt="image" src="https://github.com/filamentphp/filament/assets/1035942/eca11e55-cfc0-4f16-b1d5-f9c6a96d1663">

Expectation is to display at the end of the stack `+2`
[Documentation](https://filamentphp.com/docs/3.x/tables/columns/image#showing-the-remaining-images-count)
https://filamentphp.com/docs/3.x/tables/columns/image#showing-the-remaining-images-count

Using the latest versions:
<img width="520" alt="image" src="https://github.com/filamentphp/filament/assets/1035942/db3ab950-6a63-4f0a-8685-3d878f35acc2">
